liquid-integration/src/main/java/com/e/liquid_integration/dialog/BuyDashCryptoCurrencyDialog.kt

```diff
@@ -0,0 +1,52 @@
+package com.e.liquid_integration.dialog
+
+import android.content.Context
+import android.widget.TextView
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.e.liquid_integration.R
+import com.e.liquid_integration.`interface`.ValueSelectListner
+import com.e.liquid_integration.adapter.CryptoCurrencyAdapter
+import com.e.liquid_integration.currency.PayloadItem
+import com.google.android.material.bottomsheet.BottomSheetDialog
+import java.util.*
+
+
+class BuyDashCryptoCurrencyDialog(val _context: Context, val currencyArrayList: List<PayloadItem>, val listner: ValueSelectListner) : BottomSheetDialog(_context) {
```

Comment:
Can we rename this class to `LiquidBuyDashDialog` please? Also `_context` to `context` and `listner` to `listener`. 
Also, the class `ValueSelectListner` has a typo and should be `ValueSelectListener`.

Reply:
Fixed